service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/RequestStatementMapper.java

```diff
@@ -341,6 +343,17 @@ private String buildPriorityDescription() {
 
             return StringUtils.EMPTY;
         }
+
+        private String buildConditionalNoteDesc(Annotation value) {
+            String author = extractAuthor(messageContext, value);
+            String time = extractNoteTime(value);
+
+            if (author.equals(StringUtils.EMPTY) && time.equals(StringUtils.EMPTY)) {
```

Comment:
There is no need to compare those strings this way. You could write it this way (my recommendation):
```java
return author.isEmpty() && time.isEmpty()
	? String.format(NOTE_WITH_NO_AUTHOR_AND_TIME, value.getText())
	: String.format(NOTE, author, time, value.getText());
```
or with `if`s:
```java
if (author.isEmpty() && time.isEmpty()) {
	return String.format(NOTE_WITH_NO_AUTHOR_AND_TIME, value.getText());
} else {
	return String.format(NOTE, author, time, value.getText());
}
```